src/App.js

```diff
@@ -43,7 +56,13 @@ class App extends React.Component {
 
   return (
```

Comment:
Similarly, please check that you accept the changes where we refer to the **state object as profile** and not as userInfo.
This happens in the declaration of handleInputsOnChange and in the props of MainPage.